index.html

```diff
@@ -519,5 +703,6 @@ <h2 class="form-header">Contact me</h2>
         </section>
         <!-- Script -->
         <script src="./main.js"></script>
+        <script src="./popup.js"></script>
```

Comment:
Good job working on the details popup window 👍, however, you still need to fix some parts 

- [ ] Kindly make sure you stuck to the design as much as possible (e.g., `font`, `colors`, `images`, `text`, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃. Also make sure the popup for the mobile version is displaying well 👍 

## Figma 

> | | [For desck](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1143&scaling=min-zoom&page-id=0%3A1) (click there) | | [For mobile](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-418&scaling=min-zoom&page-id=0%3A1) (click there) | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/232258287-faa1836e-07b7-4426-ad95-9dd48aa7933b.PNG)| |![Capture](https://user-images.githubusercontent.com/57408419/232258341-e952049a-f77f-4388-8a6d-116aa18028a6.PNG)| | 

## Your design

> | |For desck| |For mobile| |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/232790426-34490c40-c715-4681-8d03-deb8773c67b6.PNG)| |![Capture](https://user-images.githubusercontent.com/57408419/232791429-e47c3e19-630f-40a1-945f-d72835ffb6fb.PNG)| |